### PR TITLE
requestSignatureVerification property added to Application Page v1.0

### PR DESCRIPTION
**Pull request by rahul-nagraj**

### CODE REVIEW

**Comment by FaithOmbongi** [comment]
Waiting for the API.md and schema PR links to be added to the content review work item.

**Comment by msewaweru** [comment]
@rahul-nagraj - the link to the schema PR takes me to the api.md while the API.md link is not the actual PR link. Please update the links on the [work item](https://dev.azure.com/msft-skilling/Content/_workitems/edit/69030/)  accordingly.

File: api-reference/v1.0/resources/requestsignatureverification.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# requestSignatureVerification resource type
+
+Namespace: microsoft.graph
+
+Specifies whether this application requires Azure AD to verify the signed authentication requests.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|allowedWeakAlgorithms|weakAlgorithms|Specifies whether this application accepts weak algorithms. <br><br> The possible values are: `rsaSha1`, `unknownFutureValue`.|
```
Comment (msewaweru) [inline api-reference/v1.0/resources/requestsignatureverification.md:19]: The description of the property can be interpreted as a boolean type.  Also, consider updating the api.md file for future references as it has a difference in properties defined.
```suggestion
|allowedWeakAlgorithms|weakAlgorithms|Specifies which weak algorithms are allowed. <br><br> The possible values are: `rsaSha1`, `unknownFutureValue`.|
```

**Review by msewaweru — CHANGES_REQUESTED** [review]
@rahul-nagraj - Thanks for the work on this PR.

 I have reviewed it and here are the suggested updates:

1. The properties for the requestsignatureVerification resource differ between the schema and the API.md. Consider updating the API.md file for future reference.
2. Remove the preview tag, if any, on docs.

**Comment by rahul-nagraj** [comment]
#feedback-addressed

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @rahul-nagraj - just one change. Please add the new property to the JSON representation section of the application.md file.

**Review by FaithOmbongi — APPROVED** [review]
Hi @rahul-nagraj - LGTM! I've only pushed one commit to change the changelog to "Applications"

Thank you @msewaweru !

**Comment by rahul-nagraj** [comment]
#sign-off

**Pull request merged**